documentation/govuk-paas.md

```diff
@@ -1,11 +1,14 @@
 # GOV.UK PaaS
 
 ## Get GOV.UK PaaS account
-Contact your organisation manager to get an account in `dfe-teacher-services` organization and in the relevant spaces:
+You will need to get an account in `dfe-teacher-services` organization and in the relevant spaces:
```

Comment:
To simplify, the new joiner should request access to the `dfe-teacher-services` org. Then the teaching vacancies tech lead or another SpaceManagers can add them to the relevant spaces (by default just dev).